packages/wallets/src/metamask/pages/login.page.ts

```diff
@@ -0,0 +1,25 @@
+import { Locator, Page, test } from '@playwright/test';
+import { WalletConfig } from '../../wallets.constants';
+
+export class LoginPage {
+  page: Page;
+  passwordInput: Locator;
+  unlockButton: Locator;
+
+  constructor(page: Page, public config: WalletConfig) {
+    this.page = page;
+    this.passwordInput = this.page.locator('id=password');
+    this.unlockButton = this.page.getByText('Unlock');
+  }
+
+  async unlock() {
+    await test.step('Unlock', async () => {
+      if (!this.page) throw "Page isn't ready";
+      if ((await this.passwordInput.count()) > 0) {
```

Comment:
why do you using this condition? Could you explain?

Reply:
It was before refactor
<img width="460" alt="image" src="https://github.com/user-attachments/assets/e82a982f-3062-4a94-9f8f-02805b933549">

I will try to change that to the `isVisible()` assert

Reply:
it is working, nice